otter/test/models/test_cass_models.py

```diff
@@ -760,6 +760,35 @@ def modifier(group, state):
         self.assertTrue(f.check(AssertionError))
         self.assertEqual(self.connection.execute.call_count, 0)
 
+    @mock.patch('otter.models.cass.CassScalingGroup.view_config',
+                return_value=defer.succeed({}))
+    def test_update_status(self, mock_vc):
+        """
+        ``update_status`` updates the status column with given named constant
+        """
+        self.clock.advance(10.345)
+        d = self.group.update_status(ScalingGroupStatus.ERROR)
+        self.assertIsNone(self.successResultOf(d))  # update returns None
+        expectedCql = ('INSERT INTO scaling_group("tenantId", "groupId", status) '
+                       'VALUES (:tenantId, :groupId, :status) USING TIMESTAMP :ts')
+        expectedData = {"status": 'ERROR',
+                        "groupId": '12345678g',
+                        "tenantId": '11111', 'ts': 10345000}
+        self.connection.execute.assert_called_with(
```

Comment:
`self.connection` is a `MagicMock` that just creates a spec with an arbitrary "execute" attribute. It may be useful to autospec/spec more precisely, to verify that these calls aren't nonsense.